jac-uk/admin
# Feature/1999 exercise contacts email dropdown

## What's included?
Closes #1999 

Note: this PR is related to [digital-platform: Get user provider data](https://github.com/jac-uk/digital-platform/pull/901)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to Contacts of an exercise and click `Update exercise contacts`.
2. Check if you can use the email dropdown to select the following fields:
    - Senior selection exercise manager
    - Selection exercise manager
    - Selection exercise officer
    - Assigned commissioner
3. Check if the fields save correctly.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/jac-uk/admin/assets/79906532/03b51a12-7ef0-4845-a40c-c81c5551cf9e

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
